### DISCUSSION TITLE
Question about sea environmental data.

### DISCUSSION THREAD

**Discussion by Kwonkyung** [MHKiT-Software/MHKiT-Python#328] [Q&A]
I am curious as to why, when using the environmental contour method, we do not only use the annual maximum data, similar to how it is applied in extreme value distributions.

Additionally, if the load value for a 100-year return period is calculated using the contour from the entire dataset, can it be directly applied to the DLC (Design Load Cases) combination?

Thank you for your assistance.

**Comment by ssolson** [comment]
Hey @Kwonkyung thank you for your interest in MHKiT!

I'm not an expert in contour methods, but here's a simple explanation. Contour methods create a regression around data, so using more data gives a better fit. If we only use annual maximum data, we can't accurately determine the probability of different sea states.

As to the DLC (Design Load Cases) combination I cannot provide insight as I am not familiar with the tool.

Hope this helps!
